frontend/src/components/regions/UserActions/UserBurndownStats.jsx

```diff
@@ -49,24 +58,32 @@ export default ({ burnDownSnapshots }) => {
             name="Total Cards"
             stroke={orange[400]}
             activeDot={{ r: 8 }}
+            strokeWidth={1}
+            dot={false}

```

Comment:
can you please include a screenshot in the pr description? I'm not sure what this will look like.